README.md

```diff
@@ -107,7 +137,11 @@ Be aware that GitHub pages has a _soft_ limit of 10 deploys per hour, and it is
 
 ## API Documentation
 
-[Please read more about how Redocusaurus is being used for API documentation.](https://transcom.github.io/mymove-docs/docs/dev/tools/redocusaurus)
+[Please read more about how Redocusaurus is being used for API documentation.](https://transcom.github.io/mymove-docs/docs/dev/tools/redocusaurus).
+
+### Updating Example Data in Documentation
+The example data in the documentation is based on swagger definitions and the behavior defined in the Service and Model. Redocly will generate examples and fill in information that is not explicitly added to swagger by default.
```

Comment:
nit: When I read the second sentence the subject is a bit unclear what redocly will generate. I think we are trying to say that without explicit example data redocly we generate and fill in some of that information. Does something like the following feel more clear?
`When example data is not provided redocly will generate examples based on the information it has available to it like the type or enum`